stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/message/Utils.kt

```diff
@@ -10,7 +10,8 @@ private val COMMAND_PATTERN = Pattern.compile("^/[a-z]*$")
 internal fun getMessageType(message: Message): String {
     val hasAttachments = message.attachments.isNotEmpty()
     val hasAttachmentsToUpload = message.attachments.any { attachment ->
-        attachment.uploadState is Attachment.UploadState.InProgress
+        attachment.uploadState is Attachment.UploadState.InProgress ||
```

Comment:
We can replace it by using extension `Message::hasPendingAttachments`

Reply:
Done.